.eslintrc.json

```diff
@@ -47,7 +47,12 @@
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error",
     "@typescript-eslint/explicit-function-return-type": "off",
-    "prettier/prettier": "error",
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
```

Comment:
Fixes an issue for developers working on Windows, where line breaks would falsely be shown as a linting error.

More info: https://stackoverflow.com/a/53769213/1843091

Reply:
Could you move your windows-specific fixes to a separate PR? I think it would be great to get them in, but they shouldn't be mixed with other bug fixes (for ease of review and isolation in case of introduced bugs).

Reply:
I think you might want to look at updating your git config instead? https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

I work on windows fairly frequently and have not had any issues with prettier. 

my machine:
```
$ git config -l
...
core.autocrlf=<boolean>
...
```

Reply:
Added in https://github.com/Uniswap/interface/pull/3730